pilot/pkg/xds/debug.go

```diff
@@ -741,6 +754,11 @@ func (s *DiscoveryServer) getConfigDumpByResourceType(conn *Connection, req *mod
 	return dumps
 }
 
+func parseAddressSlice(addr []byte) string {
+	ii, _ := netip.AddrFromSlice(addr)
+	return ii.String()
```

Comment:
Protobuf requires base64 for `bytes` - after this change we can no longer parse back to proto.

Reply:
Thanks for the reminder.